juju-charms/charm-trilio-data-mover/src/reactive/trilio_data_mover.py

```diff
@@ -91,6 +91,13 @@ def validate_nfs():
     data_dir = config('tv-data-dir')
     device = config('nfs-shares')
 
+    # install nfs-common package
+    cmd = 'apt list --installed'.split()
+    installed_list = check_output(cmd).decode('utf-8')
+    if 'nfs-common' not in installed_list:
```

Comment:
Don't we have a mechanism in charm to claim dependency on nfs-common package?  For example we check nfs-common package in our ansible scripts.
```
- name: install nfs client
  apt:
    pkg: nfs-common
    state: present
  become: true
  when: >
    ansible_distribution_release == 'xenial' and
    backup_target_type == 'NFS'
  tags:
    - configure
```

Reply:
Yes, it has a mechanism. The module 'filter_missing_packages' does the exact thing that we require. But it not documented, found this on their repo. Confirmed the usage of this module from James on IRC.

Reply:
Ok, if this is acceptable for Canonical, I am fine with  it.